packaging/rpm/ice.spec

```diff
@@ -74,14 +68,11 @@ BuildRequires: pkgconfig(mcpp)
 %endif
 
 BuildRequires: java-%{javaversion}, java-%{javaversion}-jmods
```

Comment:
We should add a comment about Java. As I understand it, we need a JDK to build the icegridgui tool. We don't build Ice for Java RPMs.